packages/workbox-runtime-caching/test/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  globals: {
```

Comment:
Personally I'm a fan of globals in eslint if all code being tested is run in that context.